src/main/java/uk/gov/digital/ho/hocs/workflow/client/documentclient/dto/DocumentDto.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.digital.ho.hocs.workflow.client.documentclient.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.LocalDateTime;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@AllArgsConstructor()
+@Getter
+@Slf4j
+public class DocumentDto {
```

Comment:
Do we need this entire DTO for the display name?

Reply:
no, I was hoping it would be useful as/when we move other calls over, but it can come in with those changes